pkg/kubectl/kubectl.go

```diff
@@ -82,6 +82,12 @@ func (e ShortcutExpander) VersionAndKindForResource(resource string) (defaultVer
 	return defaultVersion, kind, err
 }
 
+// ResourceIsValid takes a string (either group/kind or kind) and checks if it's a valid resource.
+// It expands the resource first, then invokes the wrapped mapper.
+func (e ShortcutExpander) ResourceIsValid(resource string) bool {
+	return e.RESTMapper.ResourceIsValid(expandResourceShortcut(resource))
```

Comment:
expandResourceShortcut(resource) won't work if resource is in the form of "group/kind". Currently you won't pass "group/kind" to this function, so I guess you only need to fix the comment.